olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/7.9.1/eclipse-che-preview-kubernetes.v7.9.1.clusterserviceversion.yaml.diff

```diff
@@ -0,0 +1,70 @@
+@@ -19,7 +19,7 @@
+                 "cheImageTag": "",
+                 "devfileRegistryImage": "",
+                 "pluginRegistryImage": "",
+-                "tlsSupport": false,
++                "tlsSupport": true,
+                 "selfSignedCert": false
+              },
+              "database": {
+@@ -48,13 +48,13 @@
+     capabilities: Seamless Upgrades
+     categories: Developer Tools
+     certified: "false"
+-    containerImage: quay.io/eclipse/che-operator:7.9.0
+-    createdAt: "2020-02-24T13:02:02Z"
++    containerImage: quay.io/eclipse/che-operator:7.9.1
++    createdAt: "2020-03-08T13:51:15Z"
+     description: A Kube-native development solution that delivers portable and collaborative
+       developer workspaces.
+     repository: https://github.com/eclipse/che-operator
+     support: Eclipse Foundation
+-  name: eclipse-che-preview-kubernetes.v7.9.0
++  name: eclipse-che-preview-kubernetes.v7.9.1
+   namespace: placeholder
+ spec:
+   apiservicedefinitions: {}
+@@ -248,26 +248,26 @@
+                 - name: OPERATOR_NAME
+                   value: che-operator
+                 - name: CHE_VERSION
+-                  value: 7.9.0
++                  value: 7.9.1
+                 - name: IMAGE_default_che_server
+-                  value: quay.io/eclipse/che-server:7.9.0
++                  value: quay.io/eclipse/che-server:7.9.1
+                 - name: IMAGE_default_plugin_registry
+-                  value: quay.io/eclipse/che-plugin-registry:7.9.0
++                  value: quay.io/eclipse/che-plugin-registry:7.9.1
+                 - name: IMAGE_default_devfile_registry
+-                  value: quay.io/eclipse/che-devfile-registry:7.9.0
++                  value: quay.io/eclipse/che-devfile-registry:7.9.1
+                 - name: IMAGE_default_pvc_jobs
+                   value: registry.access.redhat.com/ubi8-minimal:8.0-213
+                 - name: IMAGE_default_postgres
+                   value: centos/postgresql-96-centos7:9.6
+                 - name: IMAGE_default_keycloak
+-                  value: quay.io/eclipse/che-keycloak:7.9.0
++                  value: quay.io/eclipse/che-keycloak:7.9.1
+                 - name: IMAGE_default_che_workspace_plugin_broker_metadata
+                   value: quay.io/eclipse/che-plugin-metadata-broker:v3.1.1
+                 - name: IMAGE_default_che_workspace_plugin_broker_artifacts
+                   value: quay.io/eclipse/che-plugin-artifacts-broker:v3.1.1
+                 - name: IMAGE_default_che_server_secure_exposer_jwt_proxy_image
+-                  value: quay.io/eclipse/che-jwtproxy:dbd0578
+-                image: quay.io/eclipse/che-operator:7.9.0
++                  value: quay.io/eclipse/che-jwtproxy:810d89c
```

Comment:
this change seems LONG overdude as dbd0578 is from Sept 5, but 810d89c is from Feb 26.

You sure we shouldn't use a newer build? https://quay.io/repository/eclipse/che-jwtproxy?tag=latest&tab=tags

Reply:
https://github.com/eclipse/che/blob/7.9.1/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L505

`che.server.secure_exposer.jwtproxy.image=quay.io/eclipse/che-jwtproxy:810d89c`